apps/epic-react/schemas/documents/module.tsx

```diff
@@ -22,6 +22,7 @@ export default defineType({
       validation: (Rule) => Rule.required(),
       options: {
         list: [
+          {title: 'Module (Legacy)', value: 'legacy-module'},
```

Comment:
`legacy-module` renames the OLD module type so we can move forward with our normal naming and leave the past behind us